lib/extractSignature.js

```diff
@@ -1,11 +1,26 @@
 'use strict'
 
-const pipe = require('./pipe')
 const parseSignatureHeader = require('./parseSignatureHeader')
-const getSignature = require('./getSignature')
 
+// Public Functions
```

Comment:
I would say this and the matching `Private` comment are redundant - only the exported function is public